src/components/DatePicker/index.jsx

```diff
@@ -37,26 +37,34 @@ export default class DatePickerComponent extends React.Component {
     onChange: PropTypes.func,
     onFocus: PropTypes.func,
     query: PropTypes.object,
+    required: PropTypes.bool,
     requiredLabel: PropTypes.string,
+    showMonthDropdown: PropTypes.bool,
+    showYearDropdown: PropTypes.bool,
     title: PropTypes.string,
-    value: PropTypes.string
+    useShortMonthInDropdown: PropTypes.bool,
```

Comment:
I would rename this to showAbbreviatedDropdown maybe? I was a bit confused what shortMonth was

Reply:
Changed to `showAbbreviatedMonthDropdown`